packages/core/src/contracts/tcr/civilTCR.ts

```diff
@@ -119,8 +192,7 @@ export class CivilTCR extends BaseWrapper<CivilTCRContract> {
     return this.instance
       ._ApplicationWhitelistedStream({}, { fromBlock })
       .map(e => new Listing(this.ethApi, this.instance, e.args.listingAddress))
-      .concatMap(async l => l.getListingWrapper())
-      .concatFilter(l => isWhitelisted(l.data));
+      .concatMap(async l => l.getListingWrapper());
```

Comment:
this is no longer returning an observable that only includes currently whitelisted newsrooms..